packages/uniforms/__tests__/AutoForm.tsx

```diff
@@ -87,25 +87,6 @@ describe('AutoForm', () => {
   });
 
   describe('when rendered', () => {
-    it('calls `onChange` before render', () => {
```

Comment:
Would be best to somehow keep these tests since they are unrelated to the `autoField` prop but I don't really have an idea on how to rewrite it other than move it to a theme instead which I don't think is a good place for it. Any ideas appreciated.

Reply:
Using `AutoField.componentDetectorContext.Provider` to return the `Field` component created here would be enough.

Reply:
But `AutoField`s are also theme-specific. Can I somehow use the context provider without having one in the base package or did I misunderstand something?